extract-hackage-info.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -e
+
+WDIR=$(mktemp -d)
+HOOGLE_DATABASE="$WDIR/hoogle-database/"
+HACKAGE_DATABASE="$WDIR/hackage-database.html"
+OUTPUT="$WDIR/hackage-info.json"
+
+trap cleanup 0 1 2 3 15
+
+cleanup()
+{
+    rm -rf "$WDIR"; exit
+}
+
+nix-build -A extractHackageInfo --no-out-link
```

Comment:
I think this line should be unnecessary, as `extractHackageInfo` is built as part of `EXTRACTION_APP`?

Reply:
> I wasn't sure that only the final build output goes to stdout.

Yes, the logs go to stderr; we already use this e.g. here:

https://github.com/tweag/ormolu/blob/276dfdd9a22dfad914578984b891d2a6d33300e6/ormolu-live/deploy.sh#L3

Reply:
Fixed :+1: